velox/exec/benchmarks/CMakeLists.txt

```diff
@@ -123,3 +123,11 @@ target_link_libraries(
   velox_vector_test_lib
   velox_window
   Folly::follybenchmark)
+
+add_executable(velox_streaming_aggregation_benchmark
+               StreamingAggregationBenchmark.cpp)
+
+target_link_libraries(
+  velox_streaming_aggregation_benchmark
+  velox_exec
```

Comment:
Please be explicit about your includes and don't rely on transitive includes. Looking at the header this at least needes velox_memory but probably also other targets for the planbuilder include etc.